packages/main/src/plugin/tray-icon-colour.ts

```diff
@@ -0,0 +1,43 @@
+/**********************************************************************
+ * Copyright (C) 2022 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { ConfigurationRegistry, IConfigurationNode } from './configuration-registry';
+import type { ProviderRegistry } from './provider-registry';
+
+export class TrayIconColour {
+  constructor(private configurationRegistry: ConfigurationRegistry, private providerRegistry: ProviderRegistry) {}
+
+  async init(): Promise<void> {
+    // add configuration
+    const trayIconColourConfigurationNode: IConfigurationNode = {
```

Comment:
if it doesn't change anything on macOS, we shouldn't register that setting probably in case of macOS. WDYT ?

Reply:
I agree. Unfortunately macOS makes up their own mind for icons and we don't have ability to manually set it unless we force colour / gray into the icon.

Reply:
it's expected as we're using template images for that purpose https://www.electronjs.org/docs/latest/api/native-image#template-image that are supported only on macOS

so we don't need this dropdown/configuration on macOS

Reply:
if we really want to add the choice we should avoid to use template icons in that case here https://github.com/containers/podman-desktop/blob/main/packages/main/src/tray-animate-icon.ts#L83

but it doesn't make any sense to do that as on macOS it always pickup the good icon